sdk/edgezones/Azure.ResourceManager.EdgeZones/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Set expected release date by replacing the `Unreleased`, the format is `yyyy-mm-dd`.

Reply:
Exact release data set to 2024-04-26.